src/pages/history.tsx

```diff
@@ -41,27 +42,27 @@ import robot_2011_0 from "../images/2011photos/2011shop0.jpg";
 
 const title = "History";
 
-const data = [
+const data: HistoryCardProps[] = [
```

Comment:
Is this for type hinting with the IDE?

Reply:
Yes-- this type definition doesn't change functionality. I added it because if an item in `data` does not match `HistoryCardProps`, previously, the IDE would only show an error message when `data` is used to map to `<HistoryCard>`. Now, if there is an issue, the error message will be right on the element that does not match the interface.

I ran into this problem while modifying the array, and I found this change helpful since the entire array definition is quite long.